benchmark/slate-react/index.js

```diff
@@ -29,7 +29,7 @@ module.exports.run = function(include) {
       const dir = resolve(benchmarkDir, benchmark)
       const module = require(dir)
       const fn = module.default
-      bench.input(() => module.input)
```

Comment:
Minor change here just to make the benchmarks in `slate-react` line up better with `slate`.

The `bench.input` code already accepts either a function or a plain value, so this line was just breaking when any benchmark had `module.exports.input = () => { ... }`